core/api/usermanager/v1/usermanager_proto/usermanager.proto

```diff
@@ -16,11 +16,11 @@ syntax = "proto3";
 
 option go_package = "github.com/google/keytransparency/core/api/usermanager/v1/usermanager_proto";
 
-package keytransparency.usermanager.v1;
+package google.keytransparency.usermanager.v1;
 
 import "google/api/annotations.proto";
 import "google/protobuf/field_mask.proto";
-import "proto/keytransparency_v1_proto/keytransparency_v1_proto.proto";
+import "api/type/type_proto/type.proto";
```

Comment:
Wrong import ordering.  goimports should fix that?

Reply:
What is the correct ordering here? 
goimports doesn't work on proto files ... :-)

Reply:
If only!  There's not enough Go.

import "api/type/..."
import "google/..."